poorbook-backend/models/responses/apiResponse.py

```diff
@@ -1,57 +1,61 @@
 import sys
 sys.path.append('.')
 from pydantic import BaseModel
-from typing import Optional
 from fastapi import HTTPException
 from fastapi.exceptions import RequestValidationError
 from common.logger import InternalLogging
 from starlette import status
-from starlette.responses import Response
 from fastapi.responses import JSONResponse
+from fastapi.encoders import jsonable_encoder
 
 
 
 class ApiResponse(BaseModel):
-    """ Api endpoint response model """
-    message: str
-    status: Optional[int] = None
+    """ API Global response model """
+    message: str = ''
```

Comment:
That can be disturbing with blank default value. I would add something to tell if this hasn't changed since initialization due to status readiness after all. Any word describing like "UNKNOWN MESSAGE" or anything else is better than a blank space. Except the fact you are always taking care about the message before outputting it somewhere